SergioMendozaParra/August 26/Hackerrank/Java_Output_Formatting/Main.java

```diff
@@ -0,0 +1,36 @@
+package myPackage;
+
+import java.util.Scanner;
+
+public class Main {
+
+    public static String addZeros(int number){
+        String result;
+        if(number <= 99 && number >= 10)
+            result = "0" + number;
+        else if(number <= 9 && number >= 0)    
+            result = "00" + number;
+        else
+            result = "" + number;    
+        
+        return result;    
+    } 
+
+    public static void main(String[] args) {
+            Scanner sc=new Scanner(System.in);
+            int     number, i;
+            String  string;
+            String  spaces = "";
+            System.out.println("================================");
+            
+            for(i = 0; i < 3 ; i++){
+                string = sc.next();
+                number = sc.nextInt();
+                spaces = String.format("%18s", addZeros(number));
+                System.out.println(string + spaces.substring(string.length(), spaces.length()));    
```

Comment:
this code is somewhat obfuscated. prefer the use of **printf**

https://www.baeldung.com/java-printstream-printf

Something like this but with 2 values
![printf](https://user-images.githubusercontent.com/2263806/131264420-7f7c0d1c-3026-4f72-977f-328555f44434.png)